src/v3/plugins/github/relationalView.test.js

```diff
@@ -151,4 +151,47 @@ describe("plugins/github/relationalView", () => {
     hasCorrectParent("pull", pull);
     hasCorrectParent("review", review);
   });
+
+  describe("reference detection", () => {
+    // create url->url reference maps, for convenient snapshot readability
+    const allReferences: Map<string, Set<string>> = new Map();
```

Comment:
In the implementation, it appears possible that there could be multiple
references with a given hom. But (a) this is not the case in our example
repo (unless I’m missing something), and (b) using a `Set` loses the
ability to fully test that. I’d propose adding some multireferences and
updating the test code if needed. What do you think?